CRM/Upgrade/Incremental/sql/4.7.25.mysql.tpl

```diff
@@ -20,3 +20,7 @@ FROM civicrm_uf_join
 LEFT JOIN civicrm_event e on civicrm_uf_join.entity_id = e.id
 WHERE (civicrm_uf_join.module = 'CiviEvent' OR civicrm_uf_join.module = 'CiviEvent_Additional')
 AND e.id IS NULL;
+
+-- CRM-12167
+ALTER TABLE `civicrm_price_field_value`
+ADD COLUMN `visibility_id` int(10);
```

Comment:
As noted this needs to be moved to php but also noting it doesn't have the comment or the default value of the column